bindings/python/install_data/libtorrent/__init__.pyi

```diff
@@ -0,0 +1,1954 @@
+import datetime
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import Iterator
+from typing import List
+from typing import Mapping
+from typing import Optional
+from typing import overload
+from typing import Tuple
+from typing import Union
+
+from typing_extensions import TypedDict
+
+# I created this file by starting with stubgen and modifying its output.
+
+# I recommend that maintainers occasionally re-run stubgen, and compare it to
+# this file. This diff will be too verbose to use for small changes, but it's the
+# only way to ensure coverage.
+
+# To help with the above process, please keep everything in sorted order, to
+# match stubgen's output.
+
+# Typing notes:
+# - This project has a very particular approach to typed enum values. Pay
+#   special attention to which are typed, and which are just ints.
+# - Some attributes like add_torrent_params.save_path ought to be read as one
+#   type (str) and set as another (Union[str, bytes]), but mypy doesn't support
+#   this.
+# - We use strict primitive types (Dict and List) for function args, because
+#   boost strictly requires these, and will not understand duck-typed
+#   abstractions like Mapping or Sequence.
```

Comment:
I would think it's *possible* to essentially take `Any` by accepting `boost::python::object` on the C++ side. But it might be a bit more cumbersome to call the `__X__` functions to treat it as a dict or list.

Reply:
That would be cool. There's a downside of writing a lot of bespoke conversion code; which can become a new vector for bugs. I don't think it's a pressing issue right now.